### PR TITLE
NIFI-9316: Registry Sort by label should be 'Last Updated (newest)' not 'Newest (update)'

### PR DESCRIPTION
**Pull request by mattyb149**
Thank you for submitting a contribution to Apache NiFi.

Please provide a short description of the PR here:

#### Description of PR

Changes wording in Registry UI for clarity and consistency

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [x] Does your PR title start with **NIFI-XXXX** where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [x] Has your PR been rebased against the latest commit within the target branch (typically `main`)?

- [x] Is your initial contribution a single, squashed commit? _Additional commits in response to PR reviewer feedback should be made on this branch and pushed to allow change tracking. Do not `squash` or use `--force` when pushing to allow for clean monitoring of changes._

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed via `mvn -Pcontrib-check clean install` at the root `nifi` folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [x] Have you verified that the full build is successful on JDK 8?
- [ ] Have you verified that the full build is successful on JDK 11?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the `LICENSE` file, including the main `LICENSE` file under `nifi-assembly`?
- [ ] If applicable, have you updated the `NOTICE` file, including the main `NOTICE` file found under `nifi-assembly`?
- [ ] If adding new Properties, have you added `.displayName` in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check GitHub Actions CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/webapp/services/nf-registry.service.spec.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ describe('NfRegistry Service isolated unit tests', function () {
         expect(label).toBe('Name (a - z)');
     });
 
-    it('should get the `Newest (update)` sort by label', function () {
+    it('should get the `LastUpdated (newest)` sort by label', function () {
```
Comment (pvillard31) [inline nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/webapp/services/nf-registry.service.spec.js:71]:
```suggestion
    it('should get the `Last Updated (newest)` sort by label', function () {
```

File: nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/webapp/services/nf-registry.service.spec.js [hunk 2]
```diff
@@ -77,10 +77,10 @@ describe('NfRegistry Service isolated unit tests', function () {
         const label = nfRegistryService.getSortByLabel();
 
         //assertions
-        expect(label).toBe('Newest (update)');
+        expect(label).toBe('LastUpdated (newest)');
```
Comment (pvillard31) [inline nifi-registry/nifi-registry-core/nifi-registry-web-ui/src/main/webapp/services/nf-registry.service.spec.js:80]:
```suggestion
        expect(label).toBe('Last Updated (newest)');
```

**Review by pvillard31 — COMMENTED** [review]
I believe there is a typo, no?
Not sure why the build is green though. Is NiFi Registry included in it?

**Comment by andrewmlim** [comment]
@mattyb149 Any reason to not push this over the finish line? LGTM after Pierre's suggestions were implemented. Thanks!

**Comment by andrewmlim** [comment]
+1

**Comment by mattyb149** [comment]
After fixing the typo, getting the +1 from Andrew, and fixing the merge conflict, this one's good to go. Merging to main

**Pull request closed without merging**